### DISCUSSION TITLE
Grep case insensitive

### DISCUSSION THREAD

**Discussion by rcurran1221** [LazyVim/LazyVim#5225] [Q&A]
How do I case insensitive grep by default? How do I exit "terminal" mode?

**Comment by dpetka2001** [comment]
If you type all lower-case it should do a case insensitive search since by default fzf-lua for grep has `--smart-case` in `rg_opts`. Also if you use `FzfLua live_grep_glob` should do case-insensitive by default. `<C-\><C-n>` gets you out of terminal mode in fzf-lua picker.
